account_bank_statement_import_camt_oca/models/parser.py

```diff
@@ -130,6 +130,14 @@ def parse_entry(self, ns, node):
             ],
             transaction, 'ref'
         )
+        if self.env.context.get('journal_id'):
```

Comment:
```suggestion
        if 'journal_id' in self.env.context:
```

is more pythonic maybe